doc/docs/modules/ROOT/pages/producer-configuration.adoc

```diff
@@ -45,6 +45,12 @@ You can tune the custom property `kafka.topic.discovery.polling.interval` in ord
 periodically check for new topics into the Kafka cluster so the plugin will be able
 to send events to the defined topics.
 
+[NOTE]
+====
+To allow insertion of keys in any format (e.g. through `streams.publish` procedure) **the key.serializer is set with the org.apache.kafka.common.serialization.ByteArraySerializer**.
```

Comment:
piuttosto che così fai una sezione apposita `Serializers` in producer e scrivi li `verifica che no esista già`